downstream_projects.json

```diff
@@ -1,9 +1,8 @@
 {
   "packageDir": "./dist",
   "projects": {
-    "sample-app-angular": "https://github.com/ui-router/sample-app-angular.git",
-    "typescript48": "./test-typescript-versions/typescript4.8",
-    "angular16": "./test-angular-versions/v16",
-    "typescript49": "./test-typescript-versions/typescript4.9"
+    "sample-app-angular": "https://github.com/lindolo25/sample-app-angular.git",
```

Comment:
Change this back since I merged the other MR

Reply:
Hi @wawyed, thanks for the reply. I just added the requested change.